cli.js

```diff
@@ -36,8 +36,8 @@ program
   .option('-s, --storage-size <storage-size>', 'number of streaming files for storage purposes', Number, 10)
   .option('-p, --port <port>', 'port number the server runs on', Number, 8080)
   .option('-S, --secure', 'run with credentials for HTTPS')
-  .option('-C, --certificate <file>', 'path to SSL certificate', '')
-  .option('-K, --key <file>', 'path to private key', '')
+  .option('-C, --certificatePath <file>', 'path to SSL certificate', '')
```

Comment:
Don't change the option -- it's technically a breaking change at this point (even though this doesn't currently work). I like the old name anyways because it's shorter. The description is there to tell users that it's a path and you can assume that others would understand that as well without looking at it.

Change the variable name instead 😃